airgun/entities/ansible_role.py

```diff
@@ -39,10 +39,9 @@ def imported_roles_count(self):
         view = self.navigate_to(self, 'All')
         # Before any roles have been imported, no table or pagination widget are
         # present on the page
-        if not view.pagination.is_displayed:
-            return 0
-        else:
-            return int(view.total_imported_roles.read())
+        # Applying wait_displayed for the page to get rendered
+        view.wait_displayed()
```

Comment:
@adarshdubey-star Any specific reason to use `wait_displayed` as we can use `is_displayed` here as well.

Reply:
Yeah the issue was the page was not displayed and it returned 0 with the previous logic making the assertion fail, just gave some more time to get it rendered properly.

Reply:
This works as well.